src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -17,6 +21,13 @@
     }
   }
 
+  &__background {
```

Comment:
this should go to ContainerWithBackground, i believe.

Reply:
fixed b396ea2